File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldTextSize.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.EnumDefnTextSize;
import com.neome.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnFieldTextSize extends DefnFieldEditable
{
  @Nullable
  public MetaIdField defaultFieldId;

  @Nullable
  public EnumDefnTextSize defaultValue;

  @Nullable
  public EnumDefnTextSize defaultVar;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldTextSize.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DefnFieldEditable
import com.neome.api.meta.base.Types.EnumDefnTextSize
import com.neome.api.meta.base.Types.MetaIdField

interface DefnFieldTextSize : DefnFieldEditable
{
  val defaultFieldId: MetaIdField?
  val defaultValue: EnumDefnTextSize?
  val defaultVar: EnumDefnTextSize?
}